jest.config.js

```diff
@@ -6,7 +6,14 @@
  */
 const path = require('path');
 
-const worker = require('worker_threads');
+let worker;
+try {
+    worker = require('worker_threads');
+} catch (e) {
+    console.log(
+        'Experimental worker flag missing, skipping execution of ServiceWorker tests.'
```

Comment:
Does this actually cause the tests to skip? How does that work :o

Reply:
Yeah, referenced in my comment above, I have no idea what effect this dependency had on test execution; I see the same counts regardless if `--experimental-worker` is passed or not. Will need @revanth0212 to provide some insight.

Reply:
I remember `--experimental-worker` was need to allow node to provide `worker_threads` as a module. `worker_threads` was needed to test  Service Worker stuff. We were in the phase of mocking SW stuff instead of using the node workers, but if you are telling me that the tests are not failing even if we remove `--experimental-worker`, then maybe were have successfully mocked SW. In that case, we can just simply remove the flag.

Reply:
> Does this actually cause the tests to skip? How does that work :o

It doesn't, the tests will still run with `MessageChannel` set to `{}`.

Reply:
So there is no difference between having this code in place, and just hard-coding `MessageChannel: {}` in the globals object (just looking at counts, haven't looked at actual coverage diffs). I'm not familiar with these tests, would it better to just merge this as-is, or put in this hard-coding of `MessageChannel` and remove all the `--experimental-worker` flags?

Reply:
I would say, lets merge with this change. `MessageChannel` is used in multiple tests. Mocking it would be hard. If tests are not failing, maybe I have to update the tests to be more precise. They have to fail if `MessageChannel` is not provided or I would have mocked it already 😜